docs/developer/permissions.mdx

```diff
@@ -0,0 +1,141 @@
+---
+title: Permissions
+sidebar_label: Permissions
+---
+
+## User permissions
+
+Its group defines user permissions. Organizing access rights into [Groups](developer/api-reference/objects/group.mdx) helps in determining staff users' roles in the organization.
+
+Examples of groups:
+- Translators - `MANAGE_TRANSLATIONS` permission.
+- Customer support - `MANAGE_ORDERS` and `MANAGE_USERS` permissions.
+
+### Creating and removing groups
+To create a new group, use [permissionGroupCreate](developer/api-reference/mutations/permission-group-create.mdx).
+
+Request:
+```graphql
+mutation {
+  permissionGroupCreate(
+    input: {
+      addPermissions: [MANAGE_GIFT_CARD, MANAGE_DISCOUNTS]
+      addUsers: []
```

Comment:
Since we include this field in the example, we could also describe its behavior somewhere, for example: 
"When creating a new permission group you can optionally assign a list of members using the `addUsers` input field."

Reply:
addUsers field is explained in the managing members section, IMO there's no need to explain it twice